README.md

```diff
@@ -77,8 +87,8 @@ sudo apt-get update
 sudo apt-get install bellmansgapc
 ```
 
-## MacPorts
+### MacPorts
 
-Under Mac OS X you may want to use MacPorts to install the compiler. There is a ports description on the BiBiServ MacPorts repository. You can install GAP-C via:
+Under Mac OS X you may want to use MacPorts to install the compiler. There is a (most likely outdated) ports description on the BiBiServ MacPorts repository. You can install GAP-C via:
```

Comment:
Would it be possible to get an up-to-date ports description or even to add one ourselves?

Reply:
in principle: yes, in practice: I don't have the time :-/